### PR TITLE
Create ungoogley-worker-fired-end-retaliation-discussing-wages.en.md

### PR DESCRIPTION
**Pull request by realjenny**

### CODE REVIEW

**Comment by junefish** [comment]
why are dialogue tags bolded?

**Comment by realjenny** [comment]
I removed bolding and italics

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ 
+Workers at Alphabet have noticed an alarming trend as the workforce has shifted from majority direct employees to majority temporary, vendor or contract (TVC) staff. This has led to growing disparity in pay, benefits, and access for thousands of workers who are central to Alphabet’s functioning. Alphabet Workers Union-CWA is committed to standing in solidarity with all workers at Alphabet to ensure that our workplaces are equitable and transparent. 
+
+“We, the workers of the Alphabet Workers Union-CWA, stand in full support of Tuesday Carne and all Alphabet workers who simply want fair pay for fair work,” said Alphabet Workers Union member Rachael Sawyer, “As a TVC worker and AWU member, I am organizing to put an end to Alphabet’s segregated, two-tiered employment system. We will continue to support Tuesday, the other workers at Modis and all Alphabet workers against unjust working conditions and fear mongering from management.”
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:29]: s/,/.

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ 
+Workers at Alphabet have noticed an alarming trend as the workforce has shifted from majority direct employees to majority temporary, vendor or contract (TVC) staff. This has led to growing disparity in pay, benefits, and access for thousands of workers who are central to Alphabet’s functioning. Alphabet Workers Union-CWA is committed to standing in solidarity with all workers at Alphabet to ensure that our workplaces are equitable and transparent.
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:27]: s/contract/contractor ?

Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:27]: TVC is defined earlier in the text so could it simply be used here?

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+{{< header />}}
+
+**FOR IMMEDIATE RELEASE**
+
+December 01, 2021 at 130pm ET_
+
+_Press contact: press@alphabetworkersunion.org_
+
+*CWA Local 1400’s Alphabet Workers Union filed Unfair Labor Practice charges on Wednesday alleging that Google contractor Modis Engineering retaliated against TVC worker Tuesday Carne for protected concerted activity.*
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:17]: s/Practice/Practice (ULP)

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+date: 2021-12-01T17:00:00-08:00
+layout: textheavy
+description:
+---
+
+{{< header />}}
+
+**FOR IMMEDIATE RELEASE**
+
+December 01, 2021 at 130pm ET_
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:13]: is the time formatted correctly? should it be `1:30pm ET`

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Carne’s firing and efforts to hold Modis and Google accountable by filing an Unfair Labor Practice charge, follows a year long history of worker organizing at Modis. In February, Shannon Wait was suspended after raising concerns on behalf of herself and other workers regarding the working conditions at the Modis data center, which included a prohibition on speaking about salaries and refusal to replace damaged water bottles for workers, even as workers are expected to perform hard manual labor.  With the help of CWA Local 1400 AWU-CWA, Wait also filed a ULP against Modis and Google and [successfully won her job back](https://www.bbc.com/news/technology-56659212). In October, Modis workers were informed that they would no longer be receiving the $200 weekly attendance bonus that they had previously received for continuing to work in the midst of the COVID pandemic. Workers organized and [won back their missing payments in November](https://www.nytimes.com/2021/11/05/technology/google-workers.html).
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:25]: s/year long/year-long

Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:25]: s/are expected/were expected

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ 
+Workers at Alphabet have noticed an alarming trend as the workforce has shifted from majority direct employees to majority temporary, vendor or contract (TVC) staff. This has led to growing disparity in pay, benefits, and access for thousands of workers who are central to Alphabet’s functioning. Alphabet Workers Union-CWA is committed to standing in solidarity with all workers at Alphabet to ensure that our workplaces are equitable and transparent. 
+
+“We, the workers of the Alphabet Workers Union-CWA, stand in full support of Tuesday Carne and all Alphabet workers who simply want fair pay for fair work,” said Alphabet Workers Union member Rachael Sawyer, “As a TVC worker and AWU member, I am organizing to put an end to Alphabet’s segregated, two-tiered employment system. We will continue to support Tuesday, the other workers at Modis and all Alphabet workers against unjust working conditions and fear mongering from management.” 
+
+The unfair labor practice charge alleges that Modis management illegally retaliated against Carne for discussing wages.
```
Comment (junefish) [inline frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md:31]: s/unfair labor practice/Unfair Labor Practice 

(or just ULP?)

**Review by agdphd — APPROVED** [review]

File: frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md (hunk 5 above)
Comment (realjenny) [reply frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md]: No that should stay present tense

Comment (junefish) [reply frontend/content/press/releases/ungoogley-worker-fired-end-retaliation-discussing-wages.en.md]: isn't the refusal past tense tho

**Comment by junefish** [comment]
LGTM

**Pull request merged**